jobs/config.json

```diff
@@ -270,7 +281,8 @@
       "--extract=ci/latest-1.5",
       "--extract=ci/latest-1.6",
       "--check-leaked-resources",
-      "--check-version-skew=false"
+      "--check-version-skew=false",
+      "--test_args=None"
```

Comment:
this one seems wrong, should be `--ginkgo.focus=\[Feature:ClusterDowngrade\] --upgrade-target=ci/latest-1.5 --gce-upgrade-script=/workspace/kubernetes_skew/cluster/gce/upgrade.sh --etcd-upgrade-storage=etcd2 --etcd-upgrade-version=2.2.1`